priv/templates/extended_bin

```diff
@@ -163,14 +163,22 @@ PIPE_DIR="${PIPE_DIR:-/tmp/erl_pipes/$NAME/}"
 
 # Extract the target cookie
 COOKIE_ARG="$(grep '^-setcookie' "$VMARGS_PATH" || true)"
+DEFAULT_COOKIE_FILE="$HOME/.erlang.cookie"
 if [ -z "$COOKIE_ARG" ]; then
-    echo "vm.args needs to have a -setcookie parameter."
-    exit 1
+    if [ ! -f "$DEFAULT_COOKIE_FILE" ]; then
+        echo "vm.args needs to have a -setcookie, or $DEFAULT_COOKIE_FILE is required."
+        exit 1
+    elif [ -O "$DEFAULT_COOKIE_FILE" ] && [ $(stat --format=%a $DEFAULT_COOKIE_FILE) -eq 400 ]; then
```

Comment:
Perhaps just attempting to `cat` the file, and checking for a non-zero exit code would accomplish the same goal? If it fails, we know that it's present but unreadable by the current user, if it succeeds, we know it's clear to proceed.